dev-libs/spdlog/files/spdlog-unbundle-fmt.patch

```diff
@@ -0,0 +1,90 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index af1e398..e518a51 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -79,6 +79,7 @@ install(
+ install(
+     DIRECTORY "${HEADER_BASE}/${PROJECT_NAME}"
+     DESTINATION "${include_install_dir}"
++    PATTERN "bundled*" EXCLUDE
+ )
+ 
+ # install project version file
+diff --git a/include/spdlog/fmt/fmt.h b/include/spdlog/fmt/fmt.h
+index e2e0485..f07f8e5 100644
+--- a/include/spdlog/fmt/fmt.h
++++ b/include/spdlog/fmt/fmt.h
+@@ -10,24 +10,8 @@
+ // By default spdlog include its own copy.
+ //
+ 
+-#if !defined(SPDLOG_FMT_EXTERNAL)
+-
+-#ifndef FMT_HEADER_ONLY
+-#define FMT_HEADER_ONLY
+-#endif
+-#ifndef FMT_USE_WINDOWS_H
+-#define FMT_USE_WINDOWS_H 0
+-#endif
+-#include "bundled/format.h"
+-#if defined(SPDLOG_FMT_PRINTF)
+-#include "bundled/printf.h"
+-#endif
+-
+-#else // external fmtlib
+-
+ #include <fmt/format.h>
+ #if defined(SPDLOG_FMT_PRINTF)
+ #include <fmt/printf.h>
+ #endif
+ 
+-#endif
+diff --git a/include/spdlog/fmt/ostr.h b/include/spdlog/fmt/ostr.h
+index 9902898..c1a2223 100644
+--- a/include/spdlog/fmt/ostr.h
++++ b/include/spdlog/fmt/ostr.h
+@@ -7,12 +7,5 @@
+ //
+ // include bundled or external copy of fmtlib's ostream support
+ //
+-#if !defined(SPDLOG_FMT_EXTERNAL)
+-#ifndef FMT_HEADER_ONLY
+-#define FMT_HEADER_ONLY
+-#endif
+-#include "bundled/ostream.h"
+-#include "fmt.h"
+-#else
+ #include <fmt/ostream.h>
+-#endif
++
+diff --git a/include/spdlog/tweakme.h b/include/spdlog/tweakme.h
+index 50ad3fb..3b04a5d 100644
+--- a/include/spdlog/tweakme.h
++++ b/include/spdlog/tweakme.h
+@@ -88,13 +88,6 @@
+ // #define SPDLOG_EOL ";-)\n"
+ ///////////////////////////////////////////////////////////////////////////////
+ 
+-///////////////////////////////////////////////////////////////////////////////
+-// Uncomment to use your own copy of the fmt library instead of spdlog's copy.
+-// In this case spdlog will try to include <fmt/format.h> so set your -I flag accordingly.
+-//
+-// #define SPDLOG_FMT_EXTERNAL
```

Comment:
I think it would be cleaner to uncomment this define than removing it altogether from all files, wouldn't it?

Reply:
But this goes into `/usr/include`, doesn't it? Users are not supposed to modify files there. This whole lib looks like something you bundle with your package, not install.